Modules/xASL_module_ASL.m

```diff
@@ -213,6 +209,62 @@
 %% Define sequence (educated guess)
 x = xASL_adm_DefineASLSequence(x);
 
+%% MultiPLD parsing
+
+if isfield(x.Q,'Initial_PLD') && numel(unique(x.Q.Initial_PLD))>1  % 2 because the PLD is being repeated for control and label -> check this: 1 or 2?
```

Comment:
Great work, add more comments for the next guy that will read this!

Reply:
the numel+unique is good now, but the comment needs to be rewritten. We don't need to check for 1/2 - this has either ==1 or >1, irrespective of control and label - so please put an explanation that we go for unique PLDs.

Reply:
Done